sentry_sdk/scope.py

```diff
@@ -62,8 +67,33 @@
     T = TypeVar("T")
 
 
+# Holds data that will be added to **all** events sent by this process.
+# In case this is a http server (think web framework) with multiple users
+# the data will be added to events of all users.
+# Typically this is used for process wide data such as the release.
+_global_scope = None  # type: Optional[Scope]
+
+# Holds data for the active request.
+# This is used to isolate data for different requests or users.
+# The isolation scope is created by integrations and there should not
```

Comment:
Small spelling mistake:
```suggestion
# The isolation scope is created by integrations and they should not
```
<br />

Although, this comment contradicts what is stated in the [RFC](https://github.com/getsentry/rfcs/blob/fn/merge-hub-scope/text/0122-sdk-hub-scope-merge.md#isolation-scope):
> isolation scopes can be created, either by us internally (the most common scenario), or also by users

So, perhaps more appropriate would be to revise the comment like so:
```suggestion
# The isolation scope is usually created by integrations, but may also
```